src/frontend/js/forms/tinymce-init.ts

```diff
@@ -0,0 +1,235 @@
+            { title: "Paragraph", format: "p" },
+            { title: "Blockquote", format: "blockquote" },
+            { title: "Div", format: "div" },
+            { title: "Pre", format: "pre" },
+          ],
+        },
+        {
+          title: "Align",
+          items: [
+            { title: "Left", format: "alignleft" },
+            { title: "Center", format: "aligncenter" },
+            { title: "Right", format: "alignright" },
+            { title: "Justify", format: "alignjustify" },
+          ],
+        },
+      ],
+      min_height: 400,
+      content_css: tinymceConfig.getAttribute("data-customcss-src"),
+      language: tinymceConfig.getAttribute("data-language"),
+      setup: (editor) => {
+        editor.ui.registry.addButton("notranslate", {
+          tooltip: tinymceConfig.getAttribute("data-notranslate-tooltip"),
+          icon: "no_translate",
+          onAction: () => {
+            editor.focus();
+            const val = tinymce.activeEditor.dom.getAttrib(
+              tinymce.activeEditor.selection.getNode(),
+              "translate",
+              "yes"
+            );
+            if (val == "no") {
+              tinymce.activeEditor.dom.setAttrib(
+                tinymce.activeEditor.selection.getNode(),
+                "translate",
+                null
+              );
+            } else if (editor.selection.getContent().length > 0) {
+              editor.selection.setContent(
+                '<span class="notranslate" translate="no">' +
+                  editor.selection.getContent() +
+                  "</span>"
+              );
+            }
+          },
+        });
+        editor.ui.registry.addIcon(
+          "pin_icon",
+          parseSvg(require("../../svg/pin_icon.svg").default)
+        );
+        editor.ui.registry.addIcon(
+          "www_icon",
+          parseSvg(require("../../svg/www_icon.svg").default)
+        );
+        editor.ui.registry.addIcon(
+          "call_icon",
+          parseSvg(require("../../svg/call_icon.svg").default)
+        );
+        editor.ui.registry.addIcon(
+          "clock_icon",
+          parseSvg(require("../../svg/clock_icon.svg").default)
+        );
+        editor.ui.registry.addIcon(
+          "email_icon",
+          parseSvg(require("../../svg/email_icon.svg").default)
+        );
+        editor.ui.registry.addIcon(
+          "idea_icon",
+          parseSvg(require("../../svg/idea_icon.svg").default)
+        );
+        editor.ui.registry.addIcon(
+          "no_translate",
+          parseSvg(require("../../svg/no_translate.svg").default)
+        );
+        editor.ui.registry.addMenuItem("pinicon", {
+          text: tinymceConfig.getAttribute("data-pinicon-text"),
+          icon: "pin_icon",
+          onAction: function () {
+            editor.insertContent(
+              '<img src="' +
+                tinymceConfig.getAttribute("data-pinicon-src") +
+                '" style="width:15px; height:15px">'
```

Comment:
Nope, won't work as it actually inserts an img tag that points to that path. So the image still needs to be available in static. It would be possible to inline the svg data but that it would not be possible to switch the image later on as the actual image data would be in the database. So I'll keep it as it is right now.